src/server_manager/install_scripts/install_server.sh

```diff
@@ -68,20 +74,22 @@ function log_start_step() {
   echo -n " "
 }
 
+# Print $1 as the step name and runs the remainder as a command.
+# Forwards STDERR and logs STDERR and STDOUT.
 function run_step() {
   local -r msg=$1
-  log_start_step $msg
+  log_start_step $msg >&2
   shift 1
-  if "$@"; then
-    echo "OK"
+  if log_command "$@"; then
+    echo "OK" >&2
   else
     # Propagates the error code
     return
   fi
 }
 
 function confirm() {
-  echo -n "$1"
+  echo -n "> $1" >&2
```

Comment:
Revert

Reply:
This was deliberate, to avoid repeating the "> " prefix in each call to `confirm`.  Would prefer to repeat the prefix?

Reply:
I'm sorry, I meant the `>&2`. The prefix is fine